crates/sui-core/src/consensus_adapter.rs

```diff
@@ -398,10 +409,10 @@ impl ConsensusAdapter {
         }
 
         let delay_step = self.submit_delay_step_override.unwrap_or(latency * 2);
-
         self.metrics
-            .sequencing_estimated_latency
-            .set(latency.as_millis() as i64);
+            .sequencing_resubmission_interval_ms
+            .set(delay_step.as_millis() as i64);
```

Comment:
Do you think it would be useful to piggy back position as well to those metrics?

Reply:
Position seems independent of these metrics, so not sure about the additional value of position labels.